### PR TITLE
Use jobs names that are easier to read in the GHA UI

### PR DESCRIPTION
**Pull request by wsanchez**

### CODE REVIEW

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -150,7 +150,7 @@ jobs:
 
 
   unit:
-    name: Unit Tests using Python ${{ matrix.python-version }} on Ubuntu with Twisted ${{ matrix.twisted }}
+    name: "Twisted ${{ matrix.twisted }} on Python ${{ matrix.python-version }} on ${{ matrix.os }}"
```
Comment (adiroiban) [inline .github/workflows/cicd.yml:153]: Maybe 
```suggestion
    name: "TX ${{ matrix.twisted }} - PY ${{ matrix.python-version }} - ${{ matrix.os }}"
```

As devs should know what each means :)

**Review by adiroiban — APPROVED** [review]
Look good.

I would even replace "CI/CD" with just "CI"

The UI for master is not good... but it should get better with this merge. Thanks!

image

**Comment by wsanchez** [comment]
@adi, what do you think about:

image

**Comment by wsanchez** [comment]
It reads a bit too dense for my taste normally, but it's all visible in the UI.

**Pull request merged**